packages/standard-server-peer/src/client.test.ts

```
@@ -0,0 +1,547 @@
+          yield yieldFn('hello')
+          await new Promise(resolve => setTimeout(resolve, 100))
+          yield yieldFn('hello2')
+          yield yieldFn('hello3')
+        }
+        catch (e) {
+          iteratorError = e
+        }
+        finally {
+          isFinallyCalled = true
+          // eslint-disable-next-line no-unsafe-finally
+          throw new Error('should silence ignored')
```

Biome (1.9.4)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)